packages/commonwealth/client/scripts/stores/zustand/index.ts

```diff
@@ -0,0 +1,14 @@
+import { create } from 'zustand';
+import { createStore } from 'zustand/vanilla';
+
+import createTopicsSlice, { TopicsSlice } from './topics';
+
+// appStore can be imported and used outside the React context
+export const appStore = createStore<TopicsSlice>((...args) => ({
+  ...createTopicsSlice(...args),
```

Comment:
option 1: here we can add another slices (small stores) and then they will be accessible from single place.